Microsoft.Azure.Cosmos/src/CosmosClientBuilder.cs

```
@@ -228,6 +228,65 @@ public virtual CosmosClientBuilder UseThrottlingRetryOptions(TimeSpan maxRetryWa
             return this;
         }
 
+        /// <summary>
+        /// (Direct/TCP) Controls the amount of idle time after which unused connections are closed.
+        /// </summary>
+        /// <param name="idleTcpConnectionTimeout">By default, idle connections are kept open indefinitely. Value must be greater than or equal to 10 minutes. Recommended values are between 20 minutes and 24 hours.</param>
+        /// <remarks>
+        /// Mainly useful for sparse infrequent access to a large database account.
+        /// </remarks>
+        /// <seealso cref="CosmosClientConfiguration.IdleTcpConnectionTimeout"/>
+        public virtual CosmosClientBuilder UseIdleTcpConnectionTimeout(TimeSpan idleTcpConnectionTimeout)
```

Should it throw if the client is not configured for Direct + TCP?